eng/common/templates/steps/run-build-from-source.yml

```diff
@@ -0,0 +1,49 @@
+parameters:
+  portableOnly: 'false'
+
+steps:
+  - script: |
+      # Setup AzDo BuildConfig
+      echo "##vso[task.setvariable variable=_BuildConfig]Release"
+    displayName: Setup AzDo BuildConfig
+
+  - script: |
+      set -x
+      df -h
+
+      # Setup build arguments
+      _InternalBuildArgs=""
+      if [[ "$BUILD_REASON" != "PullRequest" && "$SYSTEM_TEAMPROJECT" != "public" ]]; then
+        _InternalBuildArgs="/p:DotNetPublishUsingPipelines=true /p:OfficialBuildId=$(BUILD.BUILDNUMBER)"
+      fi
+      if [[ "${{ parameters.portableOnly }}" == "true" ]]; then
+        _InternalBuildArgs="$_InternalBuildArgs /p:ArcadeBuildPortable=true"
+      fi
+      
+      ./build.sh --ci \
+        --configuration $(_BuildConfig) \
+        --restore --build --pack --publish \
+        $_InternalBuildArgs \
+        /p:ArcadeBuildFromSource=true > build.log
+    displayName: Build
+
+      # Upload build logs for diagnosis.
+  - task: CopyFiles@2
```

Comment:
Indent mismatch, `#` should be on same col as `-`.

Reply:
Fixed